src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1666,31 +1666,16 @@ private static SyntaxTokenList AsModifierList(Accessibility accessibility, Decla
     {
         using var _ = ArrayBuilder<SyntaxToken>.GetInstance(out var list);
 
-        switch (accessibility)
-        {
-            case Accessibility.Internal:
-                list.Add(InternalKeyword);
-                break;
-            case Accessibility.Public:
-                list.Add(PublicKeyword);
-                break;
-            case Accessibility.Private:
-                list.Add(PrivateKeyword);
-                break;
-            case Accessibility.Protected:
-                list.Add(ProtectedKeyword);
-                break;
-            case Accessibility.ProtectedOrInternal:
-                list.Add(ProtectedKeyword);
-                list.Add(InternalKeyword);
-                break;
-            case Accessibility.ProtectedAndInternal:
-                list.Add(PrivateKeyword);
-                list.Add(ProtectedKeyword);
-                break;
-            case Accessibility.NotApplicable:
-                break;
-        }
+        list.AddRange((IEnumerable<SyntaxToken>)(accessibility switch
+        {
+            Accessibility.Internal => [InternalKeyword],
```

Comment:
>Accessibility.Internal [](http://example.com/codeflow?start=12&length=22)

I don't feel too strongly as I don't know if this is called often enough to care, but the old code was about as readable to me and didn't need the allocations.

Reply:
allocations here aren't an issue.  this isn't aclled in any sort of hot loop.